tools/plcquery.py

```diff
@@ -145,6 +145,7 @@ def parse_options():
 
     parser.set_defaults(
                         action="get",
+                        session_timeout=2592000, # 30 days (60*60*24*30)
```

Comment:
This might be an argument for a global constant or preserving the numeric expression instead of using a value that isn't immediately obvious what it is on it's own.